CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Add `Registration::setFrontEndUser()` and cache it (#508)
 
 ### Changed
+- Use TYPO3 system mail as sender and current sender as reply to address in Mails (#511)
```

Comment:
This now needs to be part of the upcoming release, not of 3.0.1 anymore (which already is released).

Reply:
Oh sorry just fixed the merge conflict without looking at the version 😅

Reply:
;-)